.github/workflows/image.yaml

```diff
@@ -36,6 +36,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix: ${{fromJson(needs.get-matrix.outputs.matrix)}}
+    env:
+      - IMMUCORE_DEV: true
```

Comment:
how does this work?? is an `.env` file created by github automatically? from what I knew you could use a single env for earthly https://docs.earthly.dev/docs/guides/build-args#overriding-argument-values

Reply:
not even working locally, not my best day :D